Tests/XMTPTests/ConversationsTest.swift

```diff
@@ -122,4 +122,39 @@ class ConversationsTests: XCTestCase {
         XCTAssertFalse(Topic.isValidTopic(topic: directMessageV2))
         XCTAssertFalse(Topic.isValidTopic(topic: preferenceList))
     }
+	
+	func testLoadConvos() async throws {
```

Comment:
I wouldn't commit this test just because it uses a hardcoded wallet on dev that could change at any moment. But appreciate you thoroughly testing it.